pandas/tseries/tests/test_timeseries.py

```diff
@@ -2249,15 +2245,27 @@ def test_concat_datetime_datetime64_frame(self):
     def test_period_resample(self):
         # GH3609
         s = Series(range(100), index=date_range(
-            '20130101', freq='s', periods=100), dtype='float')
+            '20130101', freq='s', periods=100, name='idx'), dtype='float')
         s[10:30] = np.nan
-        expected = Series([34.5, 79.5], index=[Period(
-            '2013-01-01 00:00', 'T'), Period('2013-01-01 00:01', 'T')])
+        index = PeriodIndex([
+            Period('2013-01-01 00:00', 'T'),
+            Period('2013-01-01 00:01', 'T')], name='idx')
+        expected = Series([34.5, 79.5], index=index)
         result = s.to_period().resample('T', kind='period').mean()
         assert_series_equal(result, expected)
         result2 = s.resample('T', kind='period').mean()
         assert_series_equal(result2, expected)
 
+    def test_empty_period_index_resample(self):
+        # GH12771
+        index = PeriodIndex(start='2000', periods=0, freq='D', name='idx')
+        s = Series(index=index)
+        result = s.resample('M').sum().index
+
+        # would like to test vs the actual index, but due to GH12774,
+        # currently only testing the length
```

Comment:
no create the expected Series as it comes back
you can fix it later in the other issue